app/modules/cinema/endpoints_cinema.py

```diff
@@ -37,6 +41,31 @@
 hyperion_error_logger = logging.getLogger("hyperion.error")
 
 
+@module.router.get(
+    "/cinema/movie/{movie_id}",
+    response_model=schemas_cinema.Movie,
+)
+def get_movie(
+    movie_id: str,
+    settings: Settings = Depends(get_settings),
+):
+    API_key = settings.THE_MOVIE_DB_API
```

Comment:
You may want to check if the key is not None, as some Hyperion instances may decide to not include it, and return an error